### ISSUE TITLE
[QA] GUI tests sometime fails due to the system keyring dialog

### ISSUE THREAD

**Issue by ishabaral** [owncloud/client#11883]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the QA issue

Builds: https://drone.owncloud.com/owncloud/client/19378/6/11

### Screenshots

![Screenshot from 2024-09-16 14-46-22](https://github.com/user-attachments/assets/0ecd00d8-7adc-4cbc-9c75-a7f594f394cc)

![Screenshot from 2024-09-16 14-41-47](https://github.com/user-attachments/assets/e27fca7d-747c-4513-8334-52b9de862a23)

### Logs
 
[clientlog.log](https://github.com/user-attachments/files/17011373/clientlog.log)

**Comment by saw-jan** [comment]
looks like it can happen with oc10 pipeline as well
- https://drone.owncloud.com/owncloud/client/19434/5/16

The problem is with the test environment. So the server type may not be a problem

**Comment by saw-jan** [comment]
Now this appears with oc10 server pipeline :man_facepalming: 
https://drone.owncloud.com/owncloud/client/19563/2/16

**Comment by saw-jan** [comment]
Seen recently: https://drone.owncloud.com/owncloud/client/19871

**Comment by PrajwolAmatya** [comment]
Recent fails:
- https://drone.owncloud.com/owncloud/client/19922
- https://drone.owncloud.com/owncloud/client/19920
- https://drone.owncloud.com/owncloud/client/19918

**Comment by saw-jan** [comment]
yeah, the issue can appear sometime in the CI but not that frequent. So, let's move it to backlog for now.

**Comment by saw-jan** [comment]
This is still occurring these days.

**Comment by saw-jan** [comment]
keyring can be locked during the test execution
https://drone.owncloud.com/owncloud/client/20165/4/13

Trying to unlock during the test also didn't work (implemented in https://github.com/owncloud/client/pull/11866)
Maybe it takes some time to unlock?

**Comment by saw-jan** [comment]
Found out that the `gnome-keyring-daemon -r --unlock` command gets terminated sometimes in the middle of the test.

**Comment by saw-jan** [comment]
Please, reopen the issue if it occurs again
